src/main/thrift/octopus.thrift

```diff
@@ -0,0 +1,62 @@
+namespace scala com.gu.flexibleoctopus.model.thrift
+
+/** date times are reprsented as i64 - epoch millis */
+typedef i64 DateTime
+
+/** 
+* A single story bundle that maps directly to a piece of content in Composer 
+*/
+struct StoryBundle {
+
+    1: required i64 id;
+
+    2: required string composerId;
+
+    3: required string pubCode;
+
+    4: required string sectionCode;
+
+    5: required Article article;
+
+}
+
+/**
+* Represents a single article of a story bundle as generated by inCopy, 
+* such as the headline, standfirst, or body text
+*/
+struct Article {
+
+    // id for the article
+    1: required i64 id;
+
+    // id for the book section
+    2: required i64 bookSectionId;
+
+    // indicates whether the article is for web and/or print pulication
+    3: required string forPublication;
+
+    // current user that has the article checked out
+    4: required string inUseBy; 
+
+    // whether the article is currently checked out by a user
+    5: required bool isCheckedOut;
+
+    // when the last version of the article was saved
+    6: required DateTime lastModified;
+
+    // current legal status of the article
+    7: required string lawyered;
+
+    // the kind of text the article contains
+    8: required string objectType;
+
+    // the primary page that the article is on 
+    9: required i64 onPage;
```

Comment:
Do you mean the range of pages rather than a single page? We discussed with @blishen and @hoyla that we would only ever want the page that body text starts on, but that the other pages were not relevant. Is that right?

Reply:
OK - if it's been discussed and is not of value then fine 😄 
Side note: `onPage` sounds a bit like a boolean. Possibly `pageNumber` is more descriptive.